app/components/TermsCheckbox/styled.tsx

```diff
@@ -0,0 +1,10 @@
+// @ts-ignore
+import styled from 'styled-components';
+
+import Box from 'components/Box';
+
+export const TermsAndConditions = styled(Box)`
+  p {
+    margin: 10px 0;
+  }
+`;
```

Comment:
I know it's not included in this task's scope but it would be great to limit line width because too wide blocks of text effect readability and UX, see https://baymard.com/blog/line-length-readability. This text could also be justified to look better. Apply these if you like

```suggestion
export const TermsAndConditions = styled(Box)`
  max-width: 600px;
  text-align: justify;
  p {
    margin: 10px 0;
  }
`;

```